src/app/mandate.model.ts

```diff
@@ -18,6 +18,10 @@ export interface Mandate {
   giftAid: boolean,
   donationAmount: Money,
   matchedAmount: Money,
-  totalCharityReceivesPerInitial: Money,
   giftAidAmount: Money,
+  /** Includes main amount & GA but not matching */
+  totalIncGiftAid: Money,
```

Comment:
new property to be calculated in matchbot - I feel we should avoid doing any calculations (other than e.g. converting pence to pounds for display) in FE and keep all the calculations together in matchbot code.